osu.Game/Screens/Multi/Components/OverlinedDisplay.cs

```diff
@@ -83,5 +94,25 @@ private void load(OsuColour colours)
             line.Colour = colours.Yellow;
             details.Colour = colours.Yellow;
         }
+
+        private void updateDimensions()
+        {
+            grid.RowDimensions = new[]
+            {
+                new Dimension(GridSizeMode.AutoSize),
+                new Dimension(GridSizeMode.AutoSize),
+                new Dimension(AutoSizeAxes.HasFlag(Axes.Y) ? GridSizeMode.AutoSize : GridSizeMode.Distributed),
+            };
+
+            grid.AutoSizeAxes = Axes.None;
+            grid.RelativeSizeAxes = Axes.None;
+            grid.AutoSizeAxes = AutoSizeAxes;
+            grid.RelativeSizeAxes = ~AutoSizeAxes;
+
+            Content.AutoSizeAxes = Axes.None;
+            Content.RelativeSizeAxes = Axes.None;
+            Content.AutoSizeAxes = grid.AutoSizeAxes;
+            Content.RelativeSizeAxes = grid.RelativeSizeAxes;
```

Comment:
Maybe worth mentioning here (as a comment) that the `None` assignments aim to clear any sizings to make sure no exceptions about invalid sizing combinations get thrown?

Also could do compound assignment here:
```csharp
grid.AutoSizeAxes = grid.RelativeSizeAxes = Axes.None;
```
which maybe makes the intent a bit clearer but breaks the nice symmetry so it's mostly personal opinion.